tests/boardfarm_plugins/boardfarm_prplmesh/__init__.py

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
```

Comment:
Are these empty intentionally?

Reply:
> Are these empty intentionally?

Yes, all `__init__.py` were added for only purpose: mark any `*.py` file at same directory level as python module.